bin/streamr-publish.ts

```diff
@@ -1,26 +1,29 @@
-#!/usr/bin/env node
-const es = require('event-stream')
-const program = require('commander')
-const publishStream = require('../src/publish')
-const { envOptions, authOptions, exitWithHelpIfArgsNotBetween, formStreamrOptionsWithEnv } = require('./common')
+#!/usr/bin/env node -r ts-node/register
+import { Command } from 'commander';
+import es from 'event-stream'
+import { publishStream } from '../src/publish'
+import { envOptions, authOptions, exitWithHelpIfArgsNotBetween, formStreamrOptionsWithEnv } from './common'
+import pkg from '../package.json'
 
+const program = new Command();
 program
     .usage('<streamId>')
     .description('publish to a stream by reading JSON messages from stdin line-by-line')
     .option('-k, --partition-key <string>', 'field name in each message to use for assigning the message to a stream partition')
 authOptions(program)
 envOptions(program)
-    .version(require('../package.json').version)
+    .version(pkg.version)
     .parse(process.argv)
 
 exitWithHelpIfArgsNotBetween(program, 1, 1)
 
+// @ts-expect-error
```

Comment:
What are these `ts-expect-error`  about?

Reply:
Could have used casting to `any` here. 

The program in an instance of `Command` and it doesn't specify any additional options. But `formStreamrOptionsWithEnv` requires strict type of `EnvironmentOptions & AuthenticationOptions`. 

The annotation is removed in the next PR https://github.com/streamr-dev/cli-tools/pull/44 where the option parsing is updated to be compatible with Commander v7.